### DISCUSSION TITLE
Control repeat rate of :onscroll event?

### DISCUSSION THREAD

**Discussion by taylor85345** [elkowar/eww#474] [Q&A]
I'm trying to find out if it's possible to control the scroll speed/repeat rate for the :onscroll event. I'm using eww for a bar with a workspace module, and I'm trying to use :onscroll to allow scrolling through workspaces. I've got it working, but scrolling repeats too fast and a millimeter of scrolling takes me through all of my active workspaces, making it impossible to effectively control which workspace I land on.

**Comment by elkowar** [comment]
I don't think it's idiomatically possible to control that on my end, I'm afraid. You could make your script use some sort of timeout system tho, although that's of course somewhat annoying.

**Reply by taylor85345** [reply]
I'm working on a solution that will only send a workspace change request every X number of times the script is run. Was just hoping there might be a more elegant solution
